Android/app/src/main/java/io/github/project_travel_mate/VerifyEmailActivity.java

```diff
@@ -87,7 +87,7 @@ public void verifyEmail(String code, boolean verify) {
         Log.v("EXECUTING", uri);
         progressDialog.show();
 
-        //Set up client
+        //Set up Client
```

Comment:
you should do the changes applicable only for this PR only.

No need of just capitalizing a letter in file

Reply:
Sry, I didn't notice this change, all I did is editing the code in WorldClockActivity.java.
I will take a look back.